js/deleteTask.js

```diff
@@ -0,0 +1,6 @@
+export default function removeTodo(element, taskList) {
+  const index = taskList.findIndex((item) => item.id == element.id);
+  taskList[index].trash = true;
+  element.parentNode.remove();
```

Comment:
...and pass the `parentNode` of the element as the argument directly when calling the `removeTodo` function.

ex. `removeTodo(element.parentNode, taskList, element.id)`

```suggestion
  element.remove();
```